tests/integration_tests/performance/test_network_ab.py

```diff
@@ -62,25 +62,15 @@ def network_microvm(request, microvm_factory, guest_kernel, rootfs):
 
 @pytest.mark.nonci
 @pytest.mark.parametrize("network_microvm", [1], indirect=True)
-@pytest.mark.parametrize("iteration", [1, 2])
-def test_network_latency(network_microvm, metrics, iteration):
```

Comment:
I understand that with automatic re-tries we reduce the failures but does it really hurt to have more iterations? It feels like the more data we have the better.

Reply:
I can increase number of iterations in the test so that it test gives same amount of data as it was with 2 iterations. Considering each iteration gives 450 pings and I think this is enough data.

Reply:
my point was, why do we need to change this? What do we gain?

Reply:
If there are not benefits of having this test run several iterations, we should not run it several times. The original reason for this implementation is not longer relevant, so we should remove this workaround.

Reply:
Why? It does not cost us anything to have it run more times. Quite the opposite, the more iterations we run of an experiment the more statistical confidence we get for its result. We are not bound by time here, so I do not see why this is a problem.

Reply:
As I said previously, this was a workaround for A/B tests. If 450 data points is not enough for statistical confidence for a ping test, then the problem is not in the number of data points, but somewhere else. I think we should not use the logic of "the more data, the better" in this case, because this leads to the unbound upper limit on number of data we think is enough. Following this logic + as you said "we are not bound by time", why don't we increase number of iterations from 2 to 2000?

Reply:
Why leave code that brings 0 benefit and adds more confusion?

Reply:
Because it works. And it solves a problem, that of having less false positives. And it actually does it with less effort, because most likely avoids needless retries.

Reply:
It avoid needless retries by always doing the retry, just in another form. If this workaround so useful, why don't we add it to all tests. This will maybe save us even more retries.

Reply:
My main point here is that I don't like us getting in the habit of changing things that work, without causing us any problem, just for the sake of it.

Reply:
Well.. I am changing things because I work with this code at this point in time and I think it is worth to fix old hacks on the way.